vanilla-framework/vanilla-framework
# Update docs/forms page content

## Done
- Updated copy ✏️ 
- Updated demo example to showcase live example 💯 
- Check content matches copy doc [here](https://docs.google.com/document/d/1n_FNUUnBt3Mnka_c6w3lp2ov84IpC8p0Na6t7THP16s/edit#heading=h.2iegp7r10no)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/base/forms
- Pull code
- Run ./run
- Review many many many updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/form/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/checkboxes/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/disabled-input/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/fieldset/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/input/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/labels/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/select-multiple/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/radio-buttons/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/selects/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/base/forms/textarea/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-help-text/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-inline/ 
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/forms-required/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-stacked/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/forms/form-validation/

## Details
- Fixes https://github.com/ubuntudesign/design-vanilla-framework/issues/352

## Screenshots
![Screenshot 2019-04-01 at 14 16 33](https://user-images.githubusercontent.com/17748020/55330500-16209e80-5489-11e9-9da4-f662330bbbf3.png)
![Screenshot 2019-04-01 at 14 18 28](https://user-images.githubusercontent.com/17748020/55330501-16b93500-5489-11e9-9d74-224232a928d8.png)
